experimental/dds/tree/src/test/utilities/IdCompressorTestUtilities.ts

```diff
@@ -324,9 +331,29 @@ export class IdCompressorTestNetwork {
 	/**
 	 * Delivers all undelivered ID ranges and cluster capacity changes from the server to the target clients.
 	 */
-	public deliverOperations(clientTakingDelivery: DestinationClient) {
+	public deliverOperations(clientTakingDelivery: Client, opsToDeliver?: number);
+
+	/**
+	 * Delivers all undelivered ID ranges and cluster capacity changes from the server to the target clients.
+	 */
+	public deliverOperations(clientTakingDelivery: DestinationClient);
+
+	/**
+	 * Delivers all undelivered ID ranges and cluster capacity changes from the server to the target clients.
+	 */
+	public deliverOperations(clientTakingDelivery: DestinationClient, opsToDeliver?: number) {
```

Comment:
Shouldn't `clientTakingDelivery` be a `Client | DestinationClient`? Why doesn't this give a compiler error?

Reply:
Oh, because `DestinationClient` already is a `Client`, probably.